recipes/gperf/all/conanfile.py

```diff
@@ -56,6 +56,10 @@ def _configure_autotools(self):
 
     def _build_configure(self):
         with tools.chdir(self._source_subfolder):
+            try:
+                self.run("./autogen.sh")
```

Comment:
I am not sure if this is the best way to handle this please let me know if there is more standard conan way of doing this. Basically with non-released versions the `configure` script has to be generated by running `autogen.sh`.

Reply:
Instead of a try catch can we do?

```py
if file.exists() /* and self.version.startswith("cci.")*/:
    self.run(...)
```

Usually this is required

```suggestion
                self.run("autogen.sh", run_environment=True)
```

I would expect this to be with the configure autotool and do you know if this requires any args like

https://github.com/conan-io/conan-center-index/blob/b5a9d42e63a69f4fbd989152a257bda1df2d1076/recipes/gn/all/conanfile.py#L107